templates/central-configuration.yaml

```diff
@@ -149,17 +149,17 @@ spec:
             name: xl-deploy-keycloak
         {{- end }}
       {{- end }}
-  {{- if .Values.centralConfiguration.persistence.enabled }}
+  {{- if .Values.Persistence.Enabled }}
   volumeClaimTemplates:
     - metadata:
         name: data-dir
       spec:
         accessModes: [ "ReadWriteMany" ]
```

Comment:
can we use the access mode from values instead of hardcoding similar to the master/worker pods?

Reply:
Removed hardcoded accessmode